src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs

```diff
@@ -86,8 +86,7 @@ private async Task<IActionResult> GetResultAsync(HttpContext context, IFunctionE
             {
                 // Add the request to the logging scope. This allows the App Insights logger to
                 // record details about the request.
-                ILoggerFactory loggerFactory = context.RequestServices.GetService<ILoggerFactory>();
```

Comment:
Are we using the same LoggerFactory in the new code? Want to make sure there isn't a subtle change here.

Reply:
Yes, with the change, the logger factory used is that gets injected into the `ScriptHost`, which will have the same scope of what is being resolved here, so we should be good.